include/itkVkComplexToComplex1DFFTImageFilter.h

```diff
@@ -34,9 +35,9 @@ namespace itk
  *  the VkFFT library.
  *
  * This filter is multithreaded and by default supports input images with sizes which are
- * divisible by primes up to 13. 
- * 
- * Execution on input images with sizes divisible by primes greater than 17 may succeed 
+ * divisible by primes up to 13.
+ *
+ * Execution on input images with sizes divisible by primes greater than 17 may succeed
  * with a fallback on Bluestein's algorithm per VkFFT with a cost to performance and output precision.
```

Comment:
I realize that it is not part of this pull request, but:

- I would replace `divisible by primes up to 13` with something like `divisible only by primes up to 13`.
- I would replace `greater than 17` with `greater than 13`, so that it includes `17`. 

... and do the same in the two other places where this text appears.